app/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,6 @@
     <string name="enter_ifsc_code">Enter IFSC Code</string>
     <string name="bank_name">Bank Name</string>
     <string name="bank_address">Bank Address</string>
+    <string name="icon_iv">Icon_iv</string>
+    <string name="search_image_button">search_image_button</string>
```

Comment:
Why do you need this?

Reply:
The Image View and Image Button both need a 'Content Description" as, without it, they were showing warnings.